translations/round_structure.tex/pl.po

```diff
@@ -80,6 +80,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Rozgrywka podzielona jest na Rundy, podczas których każdy gracz wykonuje swoją Turę w kolejności zgodnej z ruchem wskazówek zegara, zaczynając od gracza rozpoczynającego grę.\n"
+"W swoich Turach gracze przemieszczają swoich \\hyperlink{Heroes}{Bohaterów} na Mapie Gry, budują nowe Budynki w swoich \\hyperlink{Town}{Miastach} i rekrutują \\hyperlink{Units}{Jednostki}, próbując spełnić warunek zwycięstwa Scenariusza.\\par\n"
+"\\textbf{Na początku każdej Rundy}, z wyjątkiem pierwszej, wykonaj następujące kroki :\n"
+"\\begin{itemize}\n"
+" \\item Obróć na aktywną stronę wszystkie użyte Żetony Budowy, Populacji i/lub Księgi Zaklęć.\n"
+" \\item Obróć na aktywną, zieloną stronę użyte Żetony Ruchu (PR).\n"
+" \\item Odzyskaj możliwość korzystania z Efektów Mistrzowskich  \\includesvg[height=10px]{\\svgs/expert.svg}.\n"
+"\\end{itemize}\n"
+"Następnie, w zależności od aktualnego Numeru Rundy, gracze zdobywają zasoby lub rozpatrują Kartę Astrologowie Ogłaszają:\n"
+"\\begin{itemize}\n"
+" \\item Rundy o numerach nieparzystych to \\textbf{Rundy Zasobów}.\n"
+" Wszyscy gracze otrzymują przychód z \\hyperlink{Mines}{Budynków Miasta, Osad i Kopalni}, które kontrolują.\n"
+" W pierwszej rundzie pomiń ten krok.\n"
+" \\item Rundy o numerach parzystych to \\textbf{Rundy Astrologów}.\n"
+" Dobierz Kartę Astrologowie Ogłaszają\\index{Karta Astrologowie Ogłaszają} i rozpatrz jej efekty.\n"
+" \\item Jeśli Tor Rund wskazuje na nastąpienie Wydarzenia Czasowego będącego elementem Scenariusza, rozpatrz je.\n"
```

Comment:
To trochę nie po polsku, możesz poprawić?

Reply:
"rundy parzyste" brzmi gorzej niż "rundy o numerach parzystych"
"karta astrologowie ogłaszają" to nazwa własna, wzięta bezpośrednio z instrukcji
Z ostatnim zdaniem mogę się zgodzić, z tym że ciężko jest je bezpośrednio przełożyć. Zastanawiałem się nad czymś na kształt 
(...)rozpatrz to wydarzenie."

Reply:
Chodziło tylko o linijkę 98

Reply:
Ale źle popatrzyłem, jest ok